usr/share/rear/output/default/94_grub2_rescue.sh

```diff
@@ -104,43 +104,29 @@ if [[ ! -x /etc/grub.d/01_users ]]; then
     chmod 755 /etc/grub.d/01_users
 fi
 
-if [[ $( type -f grub2-mkconfig ) ]]; then
-    grub2-mkconfig -o $grub_conf
-else
-    grub-mkconfig -o $grub_conf
-fi
-
 #Finding UUID of filesystem containing /boot 
 grub_boot_uuid=$(df /boot | awk 'END {print $1}' | xargs blkid -s UUID -o value)
```

Comment:
@schabrolles same issue here - make sure we do not get an invalid grub entry - a fallback case perhaps?